test/metric_value_benchmark/statsd_test.go

```diff
@@ -49,14 +71,56 @@ func (t *StatsdTestRunner) GetAgentRunDuration() time.Duration {
 }
 
 func (t *StatsdTestRunner) SetupAfterAgentRun() error {
-	return common.SendStatsdMetrics(2, []string{"key:value"}, time.Second, t.GetAgentRunDuration())
```

Comment:
This test got over simplified.
`common.SendStatsdMetrics()` would just send a counter and gauge type metric and the value was always 1.

I intentionally want to separate the test code in this test from the Function used by the stress test.

This test will use multiple metrics of each statsd metric type.
It will also vary the metric values so we can verify aggregation.
There is some aggregation within the input plugin and some aggregation within the output plugin.